metasrv/src/meta_service/raftmeta.rs

```diff
@@ -201,7 +203,14 @@ impl MetaNode {
         let meta_srv = RaftServiceServer::new(meta_srv_impl);
 
         let addr_str = addr.to_string();
-        let addr = addr.parse::<std::net::SocketAddr>()?;
+        let ret = addr.parse::<std::net::SocketAddr>();
+        let addr = match ret {
+            Ok(addr) => addr,
+            Err(e) => {
+                let err: MetaNetworkError = e.into();
+                return Err(err.into());
+            }
```

Comment:
If there is a double `into()`, maybe the method return value type is too general.

IMHO, a `MetaNetworkError` may be enough for this `start_grpc()` method.